src/main/scala/TelegramMessages.scala

```diff
@@ -0,0 +1,26 @@
+case class TelegramMessageFrom(
+    id: Long,
+    is_bot: Boolean,
+    first_name: String,
+    last_name: String,
+    username: String,
+    language_code: String
+)
+case class TelegramMessageChat(
+    id: Long,
+    first_name: String,
+    last_name: String,
+    username: String,
+    chat_type: String
+)
+case class TelegramMessage(
+    message_id: Long,
+    from: TelegramMessageFrom,
+    chat: TelegramMessageChat,
+    date: Long,
+    text: String
+)
+case class TelegramUpdate(
+    update_id: Long,
```

Comment:
We should use Encoder and Decoder forProduct to avoid snake cased names on scala case classes